test/models/index.js

```diff
@@ -0,0 +1,17 @@
+const models = require('../../lib/models')
+
+beforeAll(async () => {
+  if (global.gc) {
+    global.gc()
+  }
+  // Ensure there is a connection established
+  models.sequelize.authenticate()
+})
+
+// Close connection when tests are done
+afterAll(async () => models.sequelize.close())
+
+// Clear all data out of the test database
+beforeEach(() => models.sequelize.truncate({ cascade: true, restartIdentity: true }))
```

Comment:
I stole this from our slack integration to allow us to create models in our tests, and have them cleaned up.